src/config-schema.ts

```diff
@@ -18,15 +18,15 @@ const schema = {
       _type: Type.ConceptUuid,
       _default: "5086AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
     },
-    pulseUuid: {
+    heartRateUuid: {
```

Comment:
Where did these names come from? A while back I standardized them to the CIEL Fully Specified Names. e.g. https://openconceptlab.org/orgs/CIEL/sources/CIEL/concepts/5087/